features/appmgt/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/appm/js/publisher.commons.js

```diff
@@ -124,8 +124,35 @@ $(document).ready(function() {
 		$(this).next().click();
 	});
 
-	$('input[type="file"]').change(function() {
-		$(this).closest('.tbl-upload').find('.txt-filepath').val($(this).val());
+	$('input[type="file"]').change(function(evt) {
+        var acceptFileType = $(this).attr('acceptType');
+
+        if (acceptFileType.toLowerCase() == "image") {
+            var iconIdValue = $(this).attr('inputId');
+            var iconId = $("#".concat(iconIdValue));
+            var flagId= ("validity").concat(this.id);
+            var flag =  $("#".concat(flagId));
+
+            if (!flag.length) {
+               iconId.after(('<i id = "').concat(flagId).concat('"; class="icon-check-fileType"></i>'));
+               flag = $("#".concat(flagId));
+            }
+
+            var files = evt.target.files;
+            var isValidated = validateImages(files);
```

Comment:
Better to call the validateImages at the top of the if block for better readability.